src/app/components/common/data-table/data-table.component.html

```diff
@@ -54,7 +56,9 @@
                   [checked]="isRowSelected(row)"
                   (change)="toggleRowSelection(row)"
                 />
-                <label class="govuk-label govuk-checkboxes__label" [for]="row.id"></label>
+                <label class="govuk-label govuk-checkboxes__label" [for]="i">
+                  <span style="display: none">{{ 'Select checkbox ' + (row[checkboxKey] || i + 1) }}</span>
```

Comment:
Had a look at this, it doesn't seem quite right.
Row 1 & 2: "Select checkbox 1"
Row 3 & 4: "Select checkbox 2"
etc...

![image](https://github.com/hmcts/darts-portal/assets/125365979/9ee4e885-7b68-4432-9b19-56d3dac5b8d7)
![image](https://github.com/hmcts/darts-portal/assets/125365979/e78f080b-7c42-42ad-af0c-fb4d21ece6cc)